dist/web3.js

```diff
@@ -1444,6 +1444,9 @@ web3.setProvider = function (provider) {
     this.currentProvider = provider;
     RequestManager.getInstance().setProvider(provider);
 };
+web3.isConnected = function(){
```

Comment:
I'm against this method. 

- First line of [JSONRPC spec](http://www.jsonrpc.org/specification#overview) says:

  > JSON-RPC is a stateless...

  and providers should also be stateless. If a provider like IpcProvider requires persistent connection it should be handled internally inside such provider.
- also you can't tell that `HttpProvider` is `connected`. You can only say that it responded to some request.